server/controllers/memoCtrl.js

```diff
@@ -444,9 +445,15 @@ async function getAboutContent(req, res, next) {
     webContext.infoContactName = infoContactName
     webContext.examiners = examiners
 
-    webContext.memoDatas = await markOutdatedMemoDatas(rawMemos, roundInfos)
+    const memoDatas = await markOutdatedMemoDatas(rawMemos, roundInfos)
     webContext.allTypeMemos = await getMiniMemosPdfAndWeb(courseCode)
 
+    webContext.allRoundsFromKopps = await _getAllRoundsWithApplicationCodes(courseCode, responseLanguage)
+    // Adding application codes in every memo
+    // addApplicationCodesInAllTypeMemos(webContext.allTypeMemos, webContext.allRoundsFromKopps)
+    // addApplicationCodesInMemosData(memoDatas, webContext.allRoundsFromKopps)
+
+    webContext.memoDatas = memoDatas
```

Comment:
It is better to be `webContext.memoDatas = await markOutdatedMemoDatas(rawMemos, roundInfos)` two lines are replaced by one, no need in one extra variable

Reply:
done